lib/Model.ts

```diff
@@ -1,20 +1,16 @@
-'use strict';
-
-const util = require('util');
-const Q = require('q');
-const hooks = require('hooks');
-const Attribute = require('./Attribute');
-const Table = require('./Table');
-const Query = require('./Query');
-const Scan = require('./Scan');
-const Plugin = require('./Plugin');
```

Comment:
Where did this get moved to?

---

Oh I see, got moved into NewModel. Totally fine.